### PR TITLE
lutris: update to 0.4.1.

### PR DESCRIPTION
**Pull request by bra1nwave**

### CODE REVIEW

**Comment by bra1nwave** [comment]
I get this error when I try to start lutris:
```
Traceback (most recent call last):
  File "/usr/bin/lutris", line 16, in <module>
    import dbus
ImportError: No module named 'dbus'
```
`python3-dbus` is installed. Any ideas?

**Comment by ebfe** [comment]
xbps-alternatives -s python3 python3

**Comment by bra1nwave** [comment]
It works! Thank you.

**Comment by daniel-j** [comment]
Note: Lutris 0.4.0 is about to get a bug fix update, so don't merge this until it's out! https://lutris.net/releases/

**Comment by bra1nwave** [comment]
Updated to 0.4.1.

File: srcpkgs/lutris/template [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-version=0.3.8
+version=0.4.1
 revision=1
-build_style=python2-module
-pycompile_module="lutris"
+build_style=python3-module
+wrksrc="${pkgname}"
+pycompile_module="${pkgname}"
+hostmakedepends="python3-setuptools python3-gobject"
+depends="python3 python3-dbus python3-gobject python3-xdg python3-yaml
```
Comment (asergi) [inline srcpkgs/lutris/template]: `python3` is redundant here. You can remove it.
Other than that, +1.

**Comment by bra1nwave** [comment]
Removed `python3`. Thanks :)

**Pull request merged**